t/encryption.t

```diff
@@ -0,0 +1,86 @@
+use strict;
+use warnings;
+use OPCUA::Open62541 ':all';
+
+use IO::Socket::SSL::Utils;
+use MIME::Base64;
+use OPCUA::Open62541::Test::Server;
+use OPCUA::Open62541::Test::Client;
+use Test::More;
+BEGIN {
+    if(OPCUA::Open62541::ServerConfig->can('setDefaultWithSecurityPolicies')) {
+	plan tests =>
+	    OPCUA::Open62541::Test::Server::planning() +
+	    OPCUA::Open62541::Test::Client::planning() + 4;
+    } else {
+	plan skip_all => 'open62541 without UA_ENABLE_ENCRYPTION';
+    }
+}
+use Test::LeakTrace;
+use Test::NoWarnings;
+
+my %certs;
+@{$certs{server}}{qw(cert key)} = CERT_create(
+    CA => 1,
+    not_after => time() + 31536000,
```

Comment:
Please put a comment how long 31536000 is.  Or use 365\*24\*60\*60 to make the number obvious.